config/locales/en.yml

```diff
@@ -331,7 +329,7 @@ en:
       other_fields:
         property_value: "Second property or holiday home: value"
         property_mortgage: "Second property or holiday home: outstanding mortgage"
-        property_percentage_owned: "Second property or holiday home: % owned"
+        property_percentage_owned_html: "Second property or holiday home: &#37; owned"
```

Comment:
I did a bit of research and [what little I could dig up](https://stackoverflow.com/questions/20290130/how-to-deal-with-semantic-symbols-for-accessibility-in-html) suggested that in general screen readers, if faced with a symbol, will read out the name for that symbol. Which in this case would be the desired behaviour. (It might say "percent owned" or even "percent symbol owned" instead of "percentage owned", but both are accurate representations of the message on-screen.)

I'm happy to be corrected, but in the absence of obvious docs on this case, my instinct would be to wait for a broader content/accessibility review by a subject matter expert. 

In the meantime, I thought the least I could do was use a proper HTML representation of the character in question.